### PR TITLE
Nc news 6

### PR DESCRIPTION
**Pull request by jamiemharper98**

### CODE REVIEW

File: __tests__/server.test.js [hunk 1]
```diff
@@ -91,4 +93,51 @@ describe("/api", () =>
...
+});
+
+describe("/api/articles/:article_id/comments", () => {
+  test("GET:200 responds with array of comments for given article id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments.length).toBe(11);
+        expect(comments).toBeSortedBy("created_at");
```
Comment (popatre) [inline __tests__/server.test.js:130]: nice use of jest sorted

File: __tests__/server.test.js [hunk 2]
```diff
@@ -91,4 +93,51 @@ describe("/api", () =>
...
+    return request(app)
+      .get("/api/articles/999/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("No comments found!");
+      });
+  });
+  test("GET:404 responds with comments not found if correct article id has no comments", () => {
+    return request(app)
+      .get("/api/articles/2/comments")
```
Comment (popatre) [inline __tests__/server.test.js]: This should be a 200 and respond with an empty array

**Review by popatre — CHANGES_REQUESTED** [review]
Need to change your one test for an article with no comments

- [ ] Status 200: Responds with empty array if article_id is valid but has no associated comments.

**Review by nc-mick-fay — APPROVED** [review]
Great changes to reflect the feedback - merge when you're ready!

**Pull request merged**